AK/Forward.h

```diff
@@ -11,7 +11,9 @@
 namespace AK {
 
 class Bitmap;
-class ByteBuffer;
+template<size_t inline_capacity>
+class ByteBufferWithInlineCapacity;
+using ByteBuffer = ByteBufferWithInlineCapacity<32>;
```

Comment:
Isn't this a little...too excessively wordy to be in the global namespace?
maybe it could be in `AK::Detail` instead, or just the following:
```suggestion
template<size_t inline_capacity = 32>
class ByteBuffer;
```

Reply:
Aye, I've moved the class to `AK::Detail` and renamed it back to `ByteBuffer`.